react/index.js

```diff
@@ -286,7 +286,31 @@ class ProductSummary extends Component {
       'bb b--muted-4 ma2': showBorders
     })
 
-    const style = { visibility: showBuyButton ? 'visible' : 'hidden' }
+    const buyButtonStyle = { visibility: showBuyButton ? 'visible' : 'hidden' }
```

Comment:
Instead of apllying styles it should be better apply a css `class`

Reply:
`classname` ftw